ui/lib/apps/Statement/utils/tableColumns.tsx

```diff
@@ -330,27 +368,121 @@ export function statementColumns(
+    avgLocalLatchWaitColumn(tcf, rows),
+    avgWriteSizeColumn(tcf, rows),
+    avgPreWriteRegionsColumn(tcf, rows),
+    avgTxnRetryColumn(tcf, rows),
+
+    tcf.bar.single('sum_backoff_times', 'short', rows),
+    tcf.bar.single('avg_affected_rows', 'short', rows),
+
+    tcf.timestamp('first_seen'),
+    tcf.timestamp('last_seen'),
+    tcf.textWithTooltip('sample_user'),
+
+    tcf.sqlText('query_sample_text', showFullSQL),
+    tcf.sqlText('prev_sample_text', showFullSQL),
+
+    tcf.textWithTooltip('schema_name'),
+    tcf.textWithTooltip('table_names'),
+    tcf.textWithTooltip('index_names'),
+
+    tcf.textWithTooltip('plan_digest'),
+    tcf.plan('plan'),
+
+    {
+      ...tcf.textWithTooltip('related_schemas'),
+      minWidth: 160,
+      maxWidth: 240,
+    },
   ]
 }
 
 export function planColumns(rows: StatementModel[]): IColumn[] {
+  const tcf = new TableColumnFactory(TRANS_KEY_PREFIX)
+
   return [
-    planDigestColumn(rows),
-    sumLatencyColumn(rows),
-    avgMinMaxLatencyColumn(rows),
-    execCountColumn(rows),
-    avgMaxMemColumn(rows),
+    {
+      ...tcf.textWithTooltip('plan_digest'),
+      minWidth: 100,
+      maxWidth: 300,
+    },
+    tcf.bar.single('sum_latency', 'ns', rows),
+    avgMinMaxLatencyColumn(tcf, rows),
+    tcf.bar.single('exec_count', 'short', rows),
+    avgMaxMemColumn(tcf, rows),
   ]
 }
+
+////////////////////////////////////////////////
+
+export const STMT_COLUMN_REFS: { [key: string]: string[] } = {
+  avg_latency: ['avg_latency', 'min_latency', 'max_latency'],
+  avg_mem: ['avg_mem', 'max_mem'],
+  sum_errors: ['sum_errors', 'sum_warnings'],
+  avg_parse_latency: ['avg_parse_latency', 'max_parse_latency'],
+  avg_compile_latency: ['avg_compile_latency', 'max_compile_latency'],
+  avg_cop_process_time: ['avg_cop_process_time', 'max_cop_process_time'],
+  avg_cop_wait_time: ['avg_cop_wait_time', 'max_cop_wait_time'],
+  avg_process_time: ['avg_process_time', 'max_process_time'],
+
+  avg_wait_time: ['avg_wait_time', 'max_wait_time'],
+  avg_backoff_time: ['avg_backoff_time', 'max_backoff_time'],
+  avg_write_keys: ['avg_write_keys', 'max_write_keys'],
+  avg_processed_keys: ['avg_processed_keys', 'max_processed_keys'],
+  avg_total_keys: ['avg_total_keys', 'max_total_keys'],
+  avg_prewrite_time: ['avg_prewrite_time', 'max_prewrite_time'],
+  avg_commit_time: ['avg_commit_time', 'max_commit_time'],
+  avg_get_commit_ts_time: ['avg_get_commit_ts_time', 'max_get_commit_ts_time'],
+  avg_commit_backoff_time: [
+    'avg_commit_backoff_time',
+    'max_commit_backoff_time',
+  ],
+  avg_resolve_lock_time: ['avg_resolve_lock_time', 'max_resolve_lock_time'],
+  avg_local_latch_wait_time: [
+    'avg_local_latch_wait_time',
+    'max_local_latch_wait_time',
+  ],
+  avg_write_size: ['avg_write_size', 'max_write_size'],
+  avg_prewrite_regions: ['avg_prewrite_regions', 'max_prewrite_regions'],
+  avg_txn_retry: ['avg_txn_retry', 'max_txn_retry'],
+
+  related_schemas: ['table_names'],
+}
```

Comment:
Maybe we can avoid duplicating these things. These information is all available when the column is constructed.